libraries/common/subscriptions/helpers/handleLinks.js

```diff
@@ -143,11 +137,35 @@ export const isShopLink = (location) => {
   return true;
 };
 
+/**
+ * Check if the given pathname is a protector route.
+ * @param {string} location The location to check.
+ * @return {boolean}
+ */
+export const isProtector = location => Array.from(authRoutes.getAll().values()).includes(location);
+
+/**
+ * Finalizes the navigation to a legacy page or the in-app-browser.
+ * @param {string} historyAction he history action which was used to navigate.
+ */
+export const finializeNavigation = (historyAction) => {
+  const { pathname } = getCurrentRoute();
+
+  if (isProtector(pathname) && historyAction === ACTION_REPLACE) {
+    /**
+     * A replace action on a protector route indicates that the authentication was successful.
```

Comment:
it is "on a protected route". Please change.

Reply:
Within the JSDOC of the `set` method of `AuthRoutes` class the route which protects a protected route is called `protector`.

This code is about to remove the `login` page from the history when the user is redirected to an app page. So it think the term is correct here.